lib/dotcom_web/plugs/path_params_redirector.ex

```diff
@@ -0,0 +1,39 @@
+defmodule DotcomWeb.Plugs.PathParamsRedirector do
+  import Plug.Conn, only: [put_status: 2, halt: 1]
+  import Phoenix.Controller, only: [redirect: 2]
+
+  @behaviour Plug
+
+  @impl true
```

Comment:
Need to move this down. Also, I like to specify what it is implementing.

```
@impl Plug
```

Is especially useful if you ever implement two behaviors.